deployment-templates/compute-engine/client/forseti-instance-client.py

```diff
@@ -0,0 +1,157 @@
+
+    FORSETI_CLIENT_CONF = ('gs://{bucket_name}/configs/'
+                           'client/forseti_conf_client.yaml').format(
+        bucket_name=context.properties['gcs-bucket'])
+    SERVICE_ACCOUNT_SCOPES =  context.properties['service-account-scopes']
+    PERSIST_FORSETI_VARS = (
+        '\nexport FORSETI_HOME={forseti_home}\n'
+        'export FORSETI_CLIENT_CONFIG={forseti_client_conf}\n'
+        ).format(forseti_home=FORSETI_HOME,
+                 forseti_client_conf=FORSETI_CLIENT_CONF)
+
+    resources = []
+
+    resources.append({
+        'name': '{}-vm'.format(context.env['deployment']),
+        'type': 'compute.v1.instance',
+        'properties': {
+            'zone': context.properties['zone'],
+            'machineType': (
+                'https://www.googleapis.com/compute/v1/projects/{}'
+                '/zones/{}/machineTypes/{}'.format(
+                context.env['project'], context.properties['zone'],
+                context.properties['instance-type'])),
+            'disks': [{
+                'deviceName': 'boot',
+                'type': 'PERSISTENT',
+                'boot': True,
+                'autoDelete': True,
+                'initializeParams': {
+                    'sourceImage': (
+                        'https://www.googleapis.com/compute/v1'
+                        '/projects/{}/global/images/family/{}'.format(
+                            context.properties['image-project'],
+                            context.properties['image-family']
+                        )
+                    )
+                }
+            }],
+            'networkInterfaces': [{
+                'network': (
+                    'https://www.googleapis.com/compute/v1/'
+                    'projects/{}/global/networks/default'.format(
+                    context.env['project'])),
+                'accessConfigs': [{
+                    'name': 'External NAT',
+                    'type': 'ONE_TO_ONE_NAT'
+                }]
+            }],
+            'serviceAccounts': [{
+                'email': context.properties['service-account'],
+                'scopes': SERVICE_ACCOUNT_SCOPES,
+            }],
+            'metadata': {
+                'items': [{
+                    'key': 'startup-script',
+                    'value': """#!/bin/bash
+exec > /tmp/deployment.log
+exec 2>&1
+
+# Ubuntu update.
+sudo apt-get update -y
+sudo apt-get upgrade -y
+
+# Forseti setup.
+sudo apt-get install -y git unzip
+
+# Forseti dependencies
+sudo apt-get install -y libffi-dev libssl-dev libmysqlclient-dev python-pip python-dev build-essential
+
+USER=ubuntu
+USER_HOME=/home/ubuntu
+
+# Install fluentd if necessary.
+FLUENTD=$(ls /usr/sbin/google-fluentd)
+if [ -z "$FLUENTD" ]; then
+      cd $USER_HOME
+      curl -sSO https://dl.google.com/cloudagents/install-logging-agent.sh
+      bash install-logging-agent.sh
+fi
+
+# Install Forseti Security.
+cd $USER_HOME
+rm -rf *forseti*
+pip install --upgrade pip
```

Comment:
This has been changed recently and is changing in #1088 please update accordingly or find a way to factor this out of the template and into a script that's on the host, e.g. in `/setup/gcp/scripts/`